doc/requirements/srsd.rst

```diff
@@ -79,12 +200,291 @@ Functional Requirements
 
 .. req:: ARMI shall represent and reflect the evolving state of a reactor.
    :id: REQ_1
-   :status: implemented, needs test
+   :status: needs implementation, needs test
 
    The state shale be made available to users and modules, which may in turn modify the
    state (e.g. for analysis or based on the results of a physical calculation). ARMI
    shall fully define how all aspects of state may be accessed and modified and shall
    reflect any new state after it is applied.
 
-   State shall be represented as evolving eitehr through time (i.e. in a typical cycle-
+   State shall be represented as evolving either through time (i.e. in a typical cycle-
    by-cycle analysis) or through a series of control configurations.
+
+.. req:: The operator package shall provide a means by which to communicate inputs and results between analysis modules.
+   :id: REQ_OPERATOR_IO
+   :status: needs implementation, needs test
+
+The operator package shall receive output from calculation modules and store the results on a well-defined central model. A composite pattern shall be used, with a Reactor containing Assemblies containing Blocks, etc.
+
+.. req:: The operator package shall provide a means to perform computations in parallel on a high performance computer.
+   :id: REQ_OPERATOR_PARALLEL
+   :status: needs implementation, needs test
+
+Many analysis tasks require high performance computing (HPC), and the operator package shall contain utilities and routines to communicate with an HPC and to facilitate execution of simulations in parallel.
+
+.. req:: The operator package shall allow physics coupling between analysis modules.
+   :id: REQ_OPERATOR_COUPLING
+   :status: needs implementation, needs test
```

Comment:
Ah. Sure. But here "implementation" actually means referring TO the implementation in the code, by referencing this requirement in a docstring (or multiple).  Like this implementation link:

https://terrapower.github.io/armi/.apidocs/armi.reactor.grids.html#IMPL_REACTOR_MESH_0

So, in upcoming PRs I will add implementation and testing references. THEN I will mark them off as "implemented" because I have proved it.